lib/std/priority_queue.zig

```diff
@@ -77,6 +78,8 @@ pub fn PriorityQueue(comptime T: type) type {
             return if (self.len > 0) self.remove() else null;
         }
 
+        /// Remove and return the highest priority element from the
+        /// queue.
         pub fn remove(self: *Self) T {
```

Comment:
Should `remove` be changed to call `removeIdx(0)` now?

Reply:
Good point!